src/telliot_feed_examples/reporters/report_legacy_price.py

```diff
@@ -0,0 +1,122 @@
+"""Submits legacy query values to TellorX on Rinkeby."""
+import asyncio
+from typing import List
+from typing import Optional
+
+from telliot_core.apps.telliot_config import TelliotConfig
+from telliot_core.contract.contract import Contract
+from telliot_core.datafeed import DataFeed
+from telliot_core.utils.abi import rinkeby_tellor_master
+from telliot_core.utils.abi import rinkeby_tellor_oracle
+from telliot_feed_examples.feeds.btc_usd_feed import btc_usd_median_feed
+from telliot_feed_examples.feeds.eth_jpy_feed import eth_jpy_median_feed
+from telliot_feed_examples.feeds.eth_usd_feed import eth_usd_median_feed
+from telliot_feed_examples.feeds.trb_usd_feed import trb_usd_median_feed
+from telliot_feed_examples.reporters.interval import IntervalReporter
+
+
+def get_cfg() -> TelliotConfig:
+    """Get rinkeby endpoint from config
+
+    If environment variables are defined, they will override the values in config files
+    """
+    cfg = TelliotConfig()
+
+    # Override configuration for rinkeby testnet
+    cfg.main.chain_id = 4
+
+    _ = cfg.get_endpoint()
+
+    return cfg
+
+
+def get_master(cfg: TelliotConfig) -> Optional[Contract]:
```

Comment:
It seems like reporter code should not have any hard-coded rinkeby config.
Perhaps leave as-is and update after using new telliot-core directory?